utilities/helper-functions.metta

```diff
@@ -236,7 +236,30 @@
     )
 
 )
+; Function to check if an element is in a list
+(= (member $x $y)
+    (case $y
+        (
+            ( (:: $h $t) (if (== $x $h) True (member $x $t)))
+            ( () False)
+        )
+    )
+)
+
+; Function to return the intersection of two lists
```

Comment:
It's also best if you test the function whether it works with the tree data structure I gave you yesterday and let me know.

Reply:
I meant, the intersection function should work with list of tree nodes too

Reply:
Isn't this implemented already? @Yagth

Reply:
@EyobAshenaki Yeah I just saw it. But I don't think it works for a list of tree nodes. Does it?

Reply:
Yes it doesn't. Maybe it can be modified to accommodate that instead of doing it from scratch.